### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by monkrus**
## Changes

-
-
-

## Tests

```

```

## Checklist

- [x] I have read [CODE_OF_CONDUCT](https://github.com/ChainSafe/gossamer/blob/development/.github/CODE_OF_CONDUCT.md) and [CONTRIBUTING](https://github.com/ChainSafe/gossamer/blob/development/.github/CONTRIBUTING.md) 
- [x] I have provided as much information as possible and necessary
- [x] I have reviewed my own pull request before requesting a review
- [x] All integration tests and required coverage checks are passing

## Issues

-

### CODE REVIEW

File: dot/network/host_test.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ func TestConnect(t *testing.T) {
 	basePathA := utils.NewTestBasePath(t, "nodeA")
 
 	// removes all data directories created within test directory
-	defer utils.RemoveTestDir(t)
+	// defer utils.RemoveTestDir(t)
```
Comment (noot) [inline dot/network/host_test.go:32]: could you remove this, and all other commented out code?

File: dot/network/service_test.go [hunk 2]
```diff
@@ -92,7 +92,7 @@ func createTestService(t *testing.T, cfg *Config) (srvc *Service) {
 	if err != nil {
 		t.Fatal(err)
 	}
-
+	t.Cleanup(func() { utils.RemoveTestDir(t) })
```
Comment (noot) [inline dot/network/service_test.go:95]: could you also update this to include `srvc.Stop()`?

**Comment by monkrus** [comment]
Thank you for approving!

**Comment by noot** [comment]
@monkrus thank you for contributing :)

**Pull request merged**